artifacts/centos/centos.go

```diff
@@ -126,6 +128,16 @@ func getAdditionalTags(version, variant, candidate string) (additionalTags []str
 	return
 }
 
+func getImageArchitecture(arch string) string {
+	if arch == "x86_64" {
+		return "amd64"
+	} else if arch == "aarch64" {
+		return "arm64"
+	}
+
+	return "unknown"
```

Comment:
wouldn't be better to return error?

Reply:
Nope, image manifest can have `unknown` architecture.